gate-web/src/main/groovy/com/netflix/spinnaker/gate/services/BuildService.groovy

```diff
@@ -93,7 +93,7 @@ class BuildService {
       igorService.getJobConfig(buildMaster, encode(job))
     } catch (RetrofitError e) {
       if (e.response?.status == 404) {
-        throw new BuildMasterNotFound("Build master '${buildMaster}' not found")
+        throw new BuildMasterNotFound("Build master '${buildMaster}' / job '${job}' not found")
```

Comment:
`Job not found (controller: {}, job: {})`
`Builds not found (controller: {}, job: {})`
`Build not found (controller: {}, job: {}, build: {})`